tests/test_pool.py

```diff
@@ -23,13 +23,14 @@ async def getter():
 
         async with pool.acquire() as instance:
             assert instance > 0
-            await asyncio.sleep(0.01)
-            return counter
+            await asyncio.sleep(1 if counter < max_size else 0)
```

Comment:
Because I don't see the point in these tests taking 4 minutes and 27 second (266.93s) (unconditional 1 second) when they can take 3.21s instead (conditionally 1 or 0 seconds).

The longer the sleep time the more likely it is the test won't produce a false negative. 0.01 unconditionally produces false negatives on my PC because it is fast enough that it only needs to create 3 instances during the entire test.

The commit which changes that also has the same explanation in the commit message but worded differently.